frontend/src/ui/player/index.tsx

```diff
@@ -118,16 +118,38 @@ const delayTill = (date: Date): number => {
 export const InlinePlayer: React.FC<PlayerProps> = ({ className, event, ...playerProps }) => {
     const aspectRatio = getPlayerAspectRatio(event.syncedData.tracks);
     const isDark = useColorScheme().scheme === "dark";
+    const ref = useRef<HTMLDivElement>(null);
+
+    const minControlWidth = 470;
+    const [controlsFit, setControlsFit] = useState<boolean>(true);
+
+    const resizeObserver = new ResizeObserver(() => {
+        if (ref.current) {
+            const scaling = ref.current.offsetHeight / aspectRatio[1];
+            const inferredWidth = scaling * aspectRatio[0];
+            setControlsFit(inferredWidth >= minControlWidth);
+        }
+    });
+
+    useEffect(() => {
+        if (ref.current) {
+            resizeObserver.observe(ref.current);
+        }
+        return () => resizeObserver.disconnect();
+    });
 
     return (
-        <div className={className} css={{
-            "--video-container-background-color": COLORS.neutral10,
-            "--base-video-rect-background-color": COLORS.neutral10,
+        <div {...{ className }} {...{ ref }} css={{
+            "--video-container-background-color": "#101010",
+            "--base-video-rect-background-color": "#101010",
```

Comment:
Not the fault of this PR, but I just noticed: these Paella specific CSS variables should probably be set in `Paella.tsx` and not here. I mean, not that it matters right now (that we only have one player), but yeah.